Crudspec (#6)
graphqlcrud/spec: docs/spec-getOne.md

---
id: spec-getOne
title: Get Operation
sidebar_label: Get Operation
---

## Get single object 

> Fetching the note by id can be enabled by specifying id as element. 

Reviewer: ```suggestion
Fetching an object by id can be enabled by specifying id as element. 
```

> Id can be mapped to id of the object on the resolver level.

Reviewer 2: Needs to be reworded. Hard to get.

> 

Reviewer: ```suggestion
For the Note type, this can be:
```
